mne/viz/topo.py

```diff
@@ -184,12 +184,9 @@ def _plot_topo_onpick(event, show_func=None, colorbar=False):
 
 def _imshow_tfr(ax, ch_idx, tmin, tmax, vmin, vmax, ylim=None, tfr=None,
                 freq=None, vline=None, x_label=None, y_label=None,
-                colorbar=False, picker=True, cmap=None, title=None):
+                colorbar=False, picker=True, cmap='jet', title=None):
```

Comment:
FWIW I don't think it was ever used since I think all callers set this parameter themselves. I can look to see what default they use (`RdBu`?) and set it to that if you want.